package.json

```diff
@@ -17,16 +17,19 @@
     "test": "jest --detectOpenHandles",
     "lint:check": "eslint . --max-warnings=0",
     "lint:fix": "eslint --fix .",
-    "lint": "yarn lint:fix && yarn lint:check"
+    "lint": "yarn lint:fix && yarn lint:check",
+    "start-bot": "ts-node ./src/bot.ts"
   },
```

Comment:
you could consider adding start-bot to bin https://docs.npmjs.com/cli/v8/configuring-npm/package-json#bin